documentation/src/main/markdown/releasenotes.md

```diff
@@ -7,6 +7,7 @@
 ### Changed features
 
 * Added property detect.project.inspector.path to enable pointing [solution_name] to a local Project Inspector zip file.
+* Enhancements to error reporting to ensure that any exception will have the root cause reported in the error message for certain exception types. Also included in the reporting is the Black Duck response content of the transaction for api exceptions.
 
```

Comment:
To me the second sentence here ("Also included in the reporting...") feels redundant with the release note for IDETECT-3187 below. Does it add value to have both?

Reply:
In light of the fact that the PR actually says it...  I'm inclined to agree.

Reply:
Done